bin/src/learner/predict.py

```diff
@@ -0,0 +1,67 @@
+import torch
+from torch.utils.data import DataLoader
+from ..data.handlertorch import TorchDataset
+from ..utils.performance import Performance
+
+class PredictWrapper():
+    def __init__(self, model, data_path: str, experiment: object, loss_dict: dict, split: int, batch_size: int):
```

Comment:
model should also have its type descriptio. something like: 
```suggestion
    def __init__(self, model: object, data_path: str, experiment: object, loss_dict: dict, split: int, batch_size: int):
```